### PR TITLE
Shortened the test names for pylint in api sync plan tests

### PR DESCRIPTION
**Pull request by sthirugn**
pylint:
```sh
Global evaluation
-----------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```

### CODE REVIEW

File: tests/foreman/api/test_syncplan.py [hunk 1]
```diff
@@ -171,7 +170,7 @@ def test_create_sync_plan_with_random_sync_date(self):
                     sync_plan.sync_date
                 )
 
-    def test_create_sync_plan_with_invalid_name_negative(self, name):
+    def test_create_invalid_name(self, name):
         """@Test: Create a sync plan with an invalid name.
```
Comment (sthirugn) [inline tests/foreman/api/test_syncplan.py]: I figured invalid and negative are repetitive here

Comment (JacobCallahan) [inline tests/foreman/api/test_syncplan.py]: absolutely

**Comment by JacobCallahan** [comment]
APT. These changes are much more concise!

**Comment by elyezer** [comment]
ACK

**Comment by sthirugn** [comment]
Part of https://github.com/SatelliteQE/robottelo/issues/2769

**Pull request merged**